less/plugins/adapt-contrib-hotgraphic/hotgraphicPopup.less

```diff
@@ -8,17 +8,7 @@
 
   &__controls,
   &__close {
-    margin: @btn-margin / 2;
-    padding: @btn-padding / 2;
-    background-color: @notify-icon;
-    color: @notify-icon-inverted;
-    border-radius: 50%;
-
-    .no-touch &:not(.is-disabled):not(.is-locked):hover {
-      background-color: @notify-icon-hover;
-      color: @notify-icon-inverted-hover;
-      .transition(background-color @duration ease-in, color @duration ease-in;);
-    }
+    .notify-btn-icon;
   }
```

Comment:
Some small things, perhaps for another PR?

I'm not sure about using the same mixin for both the controls and the close button. I'd imagine these may want to be styled differently. For example, good UI/UX might say you don't really want to give the close button and the nav buttons equal prominence. Suggested [here](https://github.com/adaptlearning/adapt-contrib-core/issues/386)

Additionally, the close button isn't ever disabled/locked so thats also a bit confusing. 

Note: We also might want to make the border-radius a less variable? (defo a new PR)

Reply:
Valid points thanks @StuartNicholls and I agree. As the current styling of controls and close buttons uses the same variables and styles I've maintained this as the purpose of this PR was to just remove the duplication. I think border-radius and controls button styling can both be raised as new issues/PRs.

Reply:
Cool, the only thing is, do we need a different target in the templates? Fine for a separate PR here but maybe need to rationalise the classes in templates. I'll put a note [here](https://github.com/cgkineo/adapt-hotgrid/pull/127)

Reply:
Just picking this back up again. I'll merge this PR as it satisfies the [issue](https://github.com/adaptlearning/adapt-contrib-vanilla/issues/443) raised, to reduce duplicate code across plugins. I think the points made in this discussion can be tackled in separate PRs @StuartNicholls.

> I'm not sure about using the same mixin for both the controls and the close button. I'd imagine these may want to be styled differently. For example, good UI/UX might say you don't really want to give the close button and the nav buttons equal prominence. Suggested https://github.com/adaptlearning/adapt-contrib-core/issues/386

> Additionally, the close button isn't ever disabled/locked so thats also a bit confusing.

I think both of these will be part of the wider Button review in https://github.com/adaptlearning/adapt-contrib-core/issues/386

> We also might want to make the border-radius a less variable?

I'll raise a new issue off the back of this.